[2025-09-24T09:11:38.858+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_a3 manual__2025-09-24T09:09:29.387528+00:00 [queued]>
[2025-09-24T09:11:38.868+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_a3 manual__2025-09-24T09:09:29.387528+00:00 [queued]>
[2025-09-24T09:11:38.869+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-09-24T09:11:38.888+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): calculate_data_a3> on 2025-09-24 09:09:29.387528+00:00
[2025-09-24T09:11:38.897+0000] {standard_task_runner.py:60} INFO - Started process 2794 to run task
[2025-09-24T09:11:38.901+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_data_cleanse_and_calculate_all', 'calculate_data_a3', 'manual__2025-09-24T09:09:29.387528+00:00', '--job-id', '612', '--raw', '--subdir', 'DAGS_FOLDER/cleansing_data_flow.py', '--cfg-path', '/tmp/tmplk85hv67']
[2025-09-24T09:11:38.902+0000] {standard_task_runner.py:88} INFO - Job 612: Subtask calculate_data_a3
[2025-09-24T09:11:38.968+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_data_cleanse_and_calculate_all.calculate_data_a3 manual__2025-09-24T09:09:29.387528+00:00 [running]> on host feaba2204362
[2025-09-24T09:11:39.087+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_data_cleanse_and_calculate_all' AIRFLOW_CTX_TASK_ID='calculate_data_a3' AIRFLOW_CTX_EXECUTION_DATE='2025-09-24T09:09:29.387528+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-24T09:09:29.387528+00:00'
[2025-09-24T09:11:39.336+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-24T09:11:39.350+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_data_cleanse_and_calculate_all, task_id=calculate_data_a3, execution_date=20250924T090929, start_date=20250924T091138, end_date=20250924T091139
[2025-09-24T09:11:39.403+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-24T09:11:39.426+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
